fourohfour/Dockerfile

```diff
@@ -1,21 +1,27 @@
 # arxiv/fourohfour
 
-FROM arxiv/base:0.14.2
+ARG BASE_VERSION=latest
 
-WORKDIR /opt/arxiv
-
-RUN yum install -y which mysql mysql-devel
-
-ENV LOGLEVEL 10
-ENV LC_ALL en_US.utf-8
-ENV LANG en_US.utf-8
+FROM arxiv/base:${BASE_VERSION}
 
-RUN pip install -U pip pipenv uwsgi
-RUN pipenv install arxiv-base arxiv-auth
+RUN pipenv install arxiv-base arxiv-auth \
+    && rm -rf ~/.cache/pip
 
 EXPOSE 8000
 
-ADD wsgi.py uwsgi.ini entrypoint.sh /opt/arxiv/
+WORKDIR /opt/arxiv
+
+COPY wsgi.py uwsgi.ini /opt/arxiv/
+
+LABEL version="0.15.9" \
+  maintainer="arXiv <nextgen@arxiv.org>" \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="arXiv FourOhFour" \
+  org.label-schema.description="Default error backend for arXiv k8s clusters" \
+  org.label-schema.url="https://arxiv.github.io/arxiv-base" \
+  org.label-schema.vcs-url="https://github.com/arxiv/arxiv-base" \
+  org.label-schema.vendor="arXiv.org" \
+  org.label-schema.version="0.15.9"
```

Comment:
Labels that do not change from the base image should not be redeclared here.

Reply:
Fixed in 87eff32